src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/TypeAccessibilityCodeActionProvider.cs

```diff
@@ -198,10 +198,10 @@ internal class TypeAccessibilityCodeActionProvider : CSharpCodeActionProvider
             // For add using suggestions, the code action title is of the form:
             // `using System.Net;`
             else if (codeAction.Name is not null && codeAction.Name.Equals(RazorPredefinedCodeFixProviderNames.AddImport, StringComparison.Ordinal) &&
-                AddUsingsCodeActionProviderHelper.TryExtractNamespace(codeAction.Title, out var @namespace))
+                AddUsingsCodeActionProviderHelper.TryExtractNamespace(codeAction.Title, out var @namespace, out var prefix))
             {
-                codeAction.Title = $"@using {@namespace}";
-                typeAccessibilityCodeActions.Add(codeAction.WrapResolvableCodeAction(context, LanguageServerConstants.CodeActions.AddUsing));
+                codeAction.Title = $"{prefix}@using {@namespace}";
+                typeAccessibilityCodeActions.Add(codeAction.WrapResolvableCodeAction(context, LanguageServerConstants.CodeActions.Default));
```

Comment:
❓why change from "AddUsing" here?

Reply:
oh, is this because of the removal of the `AddUsingsCSharpCodeActionResolver`?

Reply:
Yep, our default resolver is capable of handling these, so no need for anything else. In fact there is an argument that we could get rid of the special provider too, it would just mean the code action would display without the `@` in them, which I suspect our users would be fine with.